datadog_checks_base/datadog_checks/base/checks/openmetrics/mixins.py

```diff
@@ -288,6 +287,25 @@ def create_scraper_configuration(self, instance=None):
         if config['metadata_metric_name'] and config['metadata_label_map']:
             config['_default_metric_transformers'][config['metadata_metric_name']] = self.transform_metadata
 
+        # Set up the HTTP wrapper for this endpoint
+
+        # TODO: Deprecate this behavior
```

Comment:
Do we know what's the condition for removing this behaviour? If any, maybe we can add it as comment.

Reply:
Added `in Agent 8`